frontend/src/pages/learn/case-studies/fred.tsx

```diff
@@ -1,23 +1,181 @@
 
+import { NavigateNext } from '@mui/icons-material'
+import { List, ListItem, ListItemButton, ListItemText, useMediaQuery } from '@mui/material'
 import { GetServerSideProps } from 'next'
-import { useTranslation } from 'next-i18next'
+import { Trans, useTranslation } from 'next-i18next'
 import { serverSideTranslations } from 'next-i18next/serverSideTranslations'
 import { NextSeo } from 'next-seo'
+import Image from 'next/image'
+import Link from 'next/link'
 
+import AccessibilityGraphContainer from '../../../components/AccessibilityGraphContainer'
+import AlertCard from '../../../components/AlertCard'
 import { LearnPageLayout } from '../../../components/LearnPageLayout'
+import theme from '../../../theme'
 import { getDCTermsTitle } from '../../../utils/seo-utils'
 
 const Fred: FC = () => {
   const { t, i18n } = useTranslation('learn/case-studies/fred')
   const en = i18n.getFixedT('en', 'learn/case-studies/fred')
   const fr = i18n.getFixedT('fr', 'learn/case-studies/fred')
 
+  let mobile = useMediaQuery(theme.breakpoints.down('sm'))
+
+  const learnMoreLinks = useMemo(
+    () => [
+      {
+        href: '/learn/deciding-when-to-collect-public-pensions',
+        primary: t('learn-more.deciding-when-to-take-your-pensions.header'),
+        secondary: t('learn-more.deciding-when-to-take-your-pensions.description'),
+      },
+      {
+        href: '/learn/rules-of-thumb-for-public-pensions',
+        primary: t('learn-more.rules-of-thumb-for-public-pensions.header'),
+        secondary: t('learn-more.rules-of-thumb-for-public-pensions.description'),
+      },
+      {
+        href: t('learn-more.oas.href'),
+        primary: t('learn-more.oas.header'),
+        secondary: t('learn-more.oas.description'),
+      },
+      {
+        href: t('learn-more.cpp.href'),
+        primary: t('learn-more.cpp.header'),
+        secondary: t('learn-more.cpp.description'),
+      },
+    ],
+    [t]
+  )
+
+  const Img = ({ pension, classes }: { pension: string, classes?: string }) => (
+    <Image
+      alt={t(`${pension}.description`)}
+      src={t(`${pension}.img-url.${mobile ? 'mobile' : 'desktop'}`)}
+      width={842}
+      height={519}
+      className={`w-full ${classes}`}
+    />
+  )
+
   return (
     <>
       <NextSeo title={t('header')} additionalMetaTags={[getDCTermsTitle(en('header'), fr('header'))]} />
-      <LearnPageLayout header={t('header')} breadcrumbItems={[]}>
-        Content here
+      <LearnPageLayout
+        header={t('header')}
+        breadcrumbItems={[
+          {
+            link: t('breadcrumbs.home.link'),
+            text: t('breadcrumbs.home.text'),
+          },
+          {
+            link: t('breadcrumbs.learn.link'),
+            text: t('breadcrumbs.learn.text'),
+          },
+        ]}
+      >
+        <NextSeo title={t('header')} />
+        
+        <h2 id="key-takeaways" className="h2 !mt-0">
+          {t('key-takeaways.heading')}
+        </h2>
+        <ul className='space-y-2'>
```

Comment:
Should use MUI List component as mentioned in PR #182

Reply:
fixed